### DISCUSSION TITLE
Capstone_ML

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#482] [Q&A] [answered]
Selamat malam
Saya ingin bertanya

Saya sedang mengerjakan capstone ML case Airline Prediction
Pada dataframe flight terdapat kolom year, month, day, hour, minute hasil break dari kolom time_hour
Sebaiknya kolom2 tersebut saya ubah menjadi factor atau datetime?

Terima kasih

**Comment by ageljonathan** [comment]
Begitu juga dengan kolom dep_time
Apakah dibiarkan menjadi numerik atau diubah menjadi factor?

**Comment by nisa-basalamah** [comment, marked as answer]
Dear @ageljonathan 

Untuk hal tersebut merupakan bagian dari eksplorasi dan disesuaikan dengan pandangan Anda terhadap isi dari masing-masing kolom tersebut apakah cocok untuk perubahan tipe data yang akan dilakukan, yang mana hal ini akan menjadi bagian dari penilaian untuk capstone ML.

Demikian yang dapat kami sampaikan, terima kasih.
